resources/results.py

```diff
@@ -11,12 +13,13 @@
 )
 
 path_to_gnomad_rf = (
-    "gs://gnomad-public-requester-pays/release/3.1/pca/gnomad.v3.1.RF_fit.pkl"
+    "gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.RF_fit.pkl"
```

Comment:
Ah yes, we have removed things in `gnomad-public-requester-pays` since they have all been synced to `gcp-public-data--gnomad`